src/planmysem/common/Utils.java

```diff
@@ -37,9 +50,129 @@ public static boolean elementsAreUnique(Collection<?> items) {
     }
 
     /**
-     * TODO
+     * Check if String represents an actual date or day.
+     * Returns 0 if string does not represent a DayOfWeek, else returns int corresponding to the day.
+     */
+    public static int getDay(String unknown) {
+        String day = unknown.toLowerCase();
+
+        int result = 0;
+        switch (day) {
+        case "monday":
+        case "mon":
+        case "1":
+            result = 1;
+            break;
+
+        case "tuesday":
+        case "tues":
+        case "2":
+            result = 2;
+            break;
+
+        case "wednesday":
+        case "wed":
+        case "3":
+            result = 3;
+            break;
+        case "thursday":
+        case "thurs":
+        case "4":
+            result = 4;
+            break;
+
+        case "friday":
+        case "fri":
+        case "5":
+            result = 5;
+            break;
+
+        default:
+            result = 0;
+        }
+
+        return result;
+    }
+
+    /**
+     * Determine recurrence if recess week.
+     */
+    public static boolean containsRecess(Set<String> recurrences) {
+        if (recurrences.contains("recess")) {
+            return true;
+        } else {
+            return false;
+        }
+    }
+
+    /**
+     * Determine recurrence if reading week.
+     */
+    public static boolean containsReading(Set<String> recurrences) {
+        if (recurrences.contains("reading")) {
+            return true;
+        } else {
+            return false;
+        }
+    }
+    /**
+     * Determine recurrence if normal academic weeks.
+     */
+    public static boolean containsNormal(Set<String> recurrences) {
+        if (recurrences.contains("normal")) {
```

Comment:
return recurrences.contains("normal")

Reply:
This and the above has led me to realize that these are unused methods that were supposed to be deleted. Thank you! Changes in next commit.